consensus/state.go

```diff
@@ -354,6 +355,29 @@ func (cs *State) OnStart() error {
 		return err
 	}
 
+	// Double Signing Risk Reduction
+	if cs.privValidator != nil && cs.config.DoubleSignCheckHeight > 0 && cs.Height > 0 {
+		valAddr, err := cs.privValidator.GetPubKey()
+		if err != nil {
+			return err
+		}
+
+		doubleSignCheckHeight := cs.config.DoubleSignCheckHeight
+		if doubleSignCheckHeight > cs.Height {
+			doubleSignCheckHeight = cs.Height
+		}
+		for i := int64(1); doubleSignCheckHeight > i; i++ {
```

Comment:
```suggestion
		for i := int64(1); i <= doubleSignCheckHeight; i++ {
```

this is more standard style